pom.xml

```diff
@@ -475,7 +475,7 @@
                 <plugin>
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
-                    <version>1.4.CODICE_1</version>
+                    <version>1.6.CODICE</version>
```

Comment:
❓ I'm guessing we were okay with @clockard's naming convention here on the version?

Reply:
Yeah that's the version that was given to our patched release of the plugin